YoutubeDownloader/ViewModels/Framework/DialogScreen.cs

```diff
@@ -1,19 +1,30 @@
 ﻿using System;
-using Stylet;
 
 namespace YoutubeDownloader.ViewModels.Framework;
 
-public abstract class DialogScreen<T> : PropertyChangedBase
+public abstract class DialogScreen<T> : ViewModelBase
 {
     public T? DialogResult { get; private set; }
 
     public event EventHandler? Closed;
 
-    public void Close(T? dialogResult = default)
+    public void Close(T? dialogResult)
     {
         DialogResult = dialogResult;
         Closed?.Invoke(this, EventArgs.Empty);
     }
+
+    // object paramter required for Avalonia binding
+    public void Close(object? dialogResult)
```

Comment:
Is this required because `T` is generic?

Reply:
same as for your comment on DashboardViewModel